block-node/stream-publisher/src/main/java/org/hiero/block/node/stream/publisher/LiveStreamPublisherManager.java

```diff
@@ -250,6 +250,7 @@ private BlockAction getActionForHeader(final long blockNumber, final long handle
      * todo(1420) add documentation
      */
     private BlockAction addHandlerQueueForBlock(final long blockNumber, final long handlerId) {
+        metrics.lowestBlockNumber.set(blockNumber);
```

Comment:
Here and below are we confident that the numbers will always match the expected lowest and highest.
Should we be checking the current is either < (lowest) or > (highest) that the previous?